CONTRIBUTING.md

```diff
@@ -281,14 +287,29 @@ Pull Request should be named in reference to the main fix or feature you provide
 >:point_right:Added a Table of Content for the Contributing Guideline Document.:point_left:
 ---
 
-### <a name="4.2.5.2"></a> 4.2.5.2 How to Write a Description for a Pull Request.
+### <a name="4.2.5.2"></a> 4.2.5.2 Before Send a Pull Request.
+
+**1 - Pull Request Description:** Write a description about the changes, we provide a [template](https://github.com/flyve-mdm/glpi-plugin/community) for Pull Request descriptions. When you're creating a Pull Request it'll be shown automatically. Just fill it out and you're done.
+
+**2 - Choose the right label**: Look at the [list of available labels.](https://github.com/flyve-mdm/glpi-plugin/issues/labels)
+
+**3 - Smash that button!** Press that _Create Pull Request_ button and you're done.
+
+**&mdash; That's it! :tada:**
+
+### <a name="4.2.5.3"></a> 4.2.5.3 How We Check your Submission.
```

Comment:
Could yo add a note about which status checks is required?

Reply:
Done!